README.md

```diff
@@ -18,6 +18,17 @@ runtime errors without sacrificing performance. It takes full advantage of
 Rust's type system to create a low overhead query builder that "feels like
 Rust."
 
+Supported databases:
+1. [PostgreSQL](https://docs.diesel.rs/diesel/pg/index.html)
+2. [MySQL](https://docs.diesel.rs/diesel/mysql/index.html)
+3. [SQLite](https://docs.diesel.rs/diesel/sqlite/index.html)
```

Comment:
Currently, it points to master branch's doc, not released one. I would prefer later but it's not a strong opinion, would like to up to other reviewers.

Reply:
We point at various places to the docs for the master branch. With merging #2269 I would suggest moving the docs from the master branch to a different location and having the docs for the latest release there, because there are quite a lot places linking there.

Reply:
So should I leave it as it is?